massa-versioning/src/versioning.rs

```diff
@@ -921,12 +922,14 @@ impl MipStoreRaw {
                 .stats
                 .network_version_counters
                 .get(&mi.version)
-                .unwrap_or(&0) as f32;
-            let block_count_considered = self.stats.config.block_count_considered as f32;
-
-            let vote_ratio_ = 100.0 * network_version_count / block_count_considered;
-
-            let vote_ratio = Amount::const_init(vote_ratio_.round() as u64, 0);
+                .unwrap_or(&0); // as f32;
+
+            let vote_ratio = Ratio::new(100, 1);
+            vote_ratio.mul(Ratio::new(network_version_count, 1));
```

Comment:
I would directly use the method `from_integer` to construct these. It probably is also possible to multiply and divided by an integer directly.

Reply:
fixed